api/spec/v1.yaml

```diff
@@ -961,6 +961,17 @@ paths:
             A filter on the evm log signatures.
             Note: The filter will only match on parsed (verified) EVM events.
           example: '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'
+        - in: query
+          name: evm_log_contract_addr_eth
+          schema:
+            type: string
```

Comment:
Do we tolerate the leading 0x? Add to the description and/or add an example value.

Reply:
up to ethCommon.HexToAddress and I didn't look :smiling_face_with_tear:

Reply:
Welp this thread was intended as the nudge to look. Anyway. I looked it up now, the 0x is optional. Can you please add that to the description and add an example value.

Reply:
it's `contract_address` now, only taking oasis format, with an example provided